rising/transforms/affine.py

```diff
@@ -17,13 +19,10 @@
 
 
 class Affine(BaseTransform):
-    def __init__(self, scale: AffineParamType = None,
-                 rotation: AffineParamType = None,
-                 translation: AffineParamType = None,
-                 matrix: torch.Tensor = None,
+    def __init__(self,
```

Comment:
What about the flag we discussed to switch between the 'naive' way (where you provide things for each dim in order) and the Pytorch way, where you provide it in xyz order independent of the actual axes order in your tensor?

Reply:
will add that with the next commit